BackendBench/scripts/dataset_filters.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
This doesn't seem to be a script

Reply:
it's not a script, though the next pr I'm adding after this is to add runtime data to things, and I'd want to do it here. Also we'll likely end up skipping more tests for the benchmark, so I think having all the filtering logic in one place would be smart.